Tests/CacheAdvanceTests/CacheAdvanceTests.swift

```diff
@@ -125,6 +125,80 @@ final class CacheAdvanceTests: XCTestCase {
         }
     }
 
+    private func prepareCacheWithIncorrectOffsetInHeader() throws -> CacheHeaderHandle {
```

Comment:
Since this is doing pretty complicated work, can you add a `///` comment to explain what this method does?

Reply:
OK. Let me add the comment. This part is copied from another existing [test](https://github.com/dfed/CacheAdvance/pull/64/files#diff-9de095b2bcc9ceaefec7d7de77b3a8f1d5b05e4ae82ba0da807b213ea076ebadL100"